examples/quilkin-filter-example/src/main.rs

```diff
@@ -73,5 +73,8 @@ impl FilterFactory for GreetFilterFactory {
 
 #[tokio::main]
 async fn main() {
-    run(vec![self::factory()].into_iter()).await.unwrap();
+    let log = quilkin::proxy::logger();
+    let config = quilkin::config::Config::find(&log, None).map(Arc::new)?;
+
+    run(log, config, vec![self::factory()].into_iter()).await.unwrap();
```

Comment:
I'd prefer if we had a oneline version of this so that a user that doesn't have any custom code doesn't need to know about setup bolierplate. I'm thinking similar to the previous `run` that internally creates a logger and finds the config